lobby-db-dao/src/main/java/org/triplea/lobby/server/db/ModeratorAuditHistoryItem.java

```diff
@@ -0,0 +1,45 @@
+package org.triplea.lobby.server.db;
+
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.Calendar;
+import java.util.TimeZone;
+
+import org.jdbi.v3.core.mapper.RowMapper;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+/**
+ * Bean class for transport between JDBI and http-server. Represents one row with
+ */
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+public class ModeratorAuditHistoryItem {
+  private Instant dateCreated;
+  private String username;
+  private String actionName;
+  private String actionTarget;
+
+  /**
+   * Returns a JDBI row mapper used to convert results into an instance of this bean object.
+   */
+  static RowMapper<ModeratorAuditHistoryItem> moderatorAuditHistoryItemMapper() {
+    return (rs, ctx) -> {
+      final Calendar cal = Calendar.getInstance();
+      cal.setTimeZone(TimeZone.getTimeZone("UTC"));
```

Comment:
`java.util` time classes are evil 😛 
Is there anything speaking against just omitting the timezone and converting the Timestamp straight to an Instant?
The latter isn't even supposed to contain any Timezone information anyways

Reply:
While Instant does not contain timezones, the source data for converting to an instant does matter.

Showing UTC timestamps IMO is a good thing, I don't think we should necessarily rely on DB implementation details to know that the timezone will be a UTC (I'm not sure if java.sql.ResultSet will guarantee that).

My thoughts here are:
- while java.util classes are evil, this code is 'safe' and is pretty guaranteed to give us a consistent time stamp with proper time zone. If that means 2 lines of ugly java.util code, I don't really care as long as it is more certain to be correct.

Some proper experimentation would be useful to really know the answer, but TBH i"m not willing to put that time in if the cost is a few lines of ugly java.util time, I'll trade certainly correct over the prettiest code and save time on this.